### PR TITLE
8253812: Cleanup AbstractMemberWriter

### PR DESCRIPTION
**Pull request by jonathan-gibbons**
This is primarily a cosmetic cleanup for the doc comments on `AbstractMemberWriter`, but cleaning up some of the comments leaked into fixing up comments on overridden methods as well.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8253812](https://bugs.openjdk.java.net/browse/JDK-8253812): Cleanup AbstractMemberWriter

### Reviewers
 * [Kumar Srinivasan](https://openjdk.java.net/census#ksrini) (@kusrinivasan - **Reviewer**) ⚠️ Review applies to 835e70d01c9f1a5638597d8ec33d72fa7402eb54

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/405/head:pull/405`
`$ git checkout pull/405`

### CODE REVIEW

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java [hunk 1]
```diff
@@ -300,69 +288,65 @@ protected String name(Element member) {
...
      */
-    protected void addUseInfo(List<? extends Element> mems, Content heading, Content contentTree) {
-        if (mems == null || mems.isEmpty()) {
+    protected void addUseInfo(List<? extends Element> members, Content heading, Content contentTree) {
+        if (members == null || members.isEmpty()) {
             return;
         }
-        List<? extends Element> members = mems;
         boolean printedUseTableHeader = false;
-        if (members.size() > 0) {
```
Comment (jonathan-gibbons) [inline src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java:330]: The enclosing `if` statement is unnecessary.

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java [hunk 2]
```diff
@@ -371,16 +355,9 @@ protected void serialWarning(Element e, String key, String a1, String a2) {
...
-     * Add the member summary for the given class.
-     *
-     * @param tElement the class that is being documented
-     * @param member the member being documented
-     * @param firstSentenceTags the first sentence tags to be added to the summary
-     */
     @Override
     public void addMemberSummary(TypeElement tElement, Element member,
-            List<? extends DocTree> firstSentenceTags) {
+            List<? extends DocTree> firstSentenceTrees) {
```
Comment (jonathan-gibbons) [inline src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java:360]: This use of the word `tags` is a hangover from an older version of javadoc

**Review by ksrinifmt — APPROVED** [review]
Looks good.

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java [hunk 3]
```diff
@@ -225,24 +220,19 @@ protected void addModifierAndType(Element member, TypeMirror type,
...
         tdSummaryType.add(code);
     }
 
     /**
-     * Add the modifier for the member.
+     * Adds the modifier for the member.
      *
      * @param member the member to add the type for
-     * @param code the content tree to which the modified will be added
+     * @param code   the content tree to which the modified will be added
```
Comment (kusrinivasan) [inline src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java:235]: I think this should be modifier not modified.

**Review by kusrinivasan — APPROVED** [review]

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java (hunk 3 above)
Comment (jonathan-gibbons) [reply src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java:235]: Indeed; will fix

**Pull request closed without merging**